src/toffy/watcher_callbacks.py

```diff
@@ -159,17 +167,36 @@ def _generate_fov_data(
             replace=replace,
             time_res=time_res,
         )
-
         self.__panel = panel
 
-    def extract_tiffs(self, tiff_out_dir: str, panel: pd.DataFrame, **kwargs):
+        if extract_prof:
+            # modify from (-0.3, 0) to (0, 0.3) for proficient
+            panel_prof = modify_panel_ranges(panel, start_offset=0.3, stop_offset=0.3)
```

Comment:
Why are these the same?

Reply:
The offset args define the amount that needs to get added to the Start and Stop columns respectively, not the final offset. So in this case, we need to add 0.3 to the Start column and the Stop column to ensure the range goes from (-0.3, 0) to (0, 0.3).

Reply:
I think it would be more intuitive to be able to set the `start_offset = -0.3` and the `stop_offset = 0.0`.

Reply:
I agree it's not the most intuitive, however it will be problematic to use `start_offset=-0.3` and `stop_offset=0.0`, since by default the panel will be specified in the (-0.3, 0) range.

This function is not one that should ever get called by the user. The comment is not that intuitive, so I will adjust it so it's clear that the offsets are added to the `'Start'` and `'Stop'` columns.